src/legacy/ui/public/visualize/loader/embedded_visualize_handler.ts

```diff
@@ -71,7 +71,7 @@ export class EmbeddedVisualizeHandler {
    * This should not be used by any plugin.
    * @ignore
    */
-  public static readonly __ENABLE_PIPELINE_DATA_LOADER__: boolean = false;
+  public static readonly __ENABLE_PIPELINE_DATA_LOADER__: boolean = true;
```

Comment:
Maybe it's time we made this a yml file variable so we can turn it on/off locally and for the tests without flipping the switch for everyone.  What do you think?  Or do you feel this is ready to go for 7.1?

Reply:
for this PR i am gonna turn it off, but i think we should switch it on for 7.1 and master branch